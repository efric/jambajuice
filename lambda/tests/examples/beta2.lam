(\x . (\x . x) x) one

-- should reduce to one
