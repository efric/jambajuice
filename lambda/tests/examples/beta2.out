(\x . (\x . x) x) one
